tools/distpartitioning/parmetis_preprocess.py

```diff
@@ -26,17 +26,18 @@ def get_proc_info():
     MPI_WORLDSIZE appropriately as described above to retrieve total no. of
     processes, when needed.
 
+    MPI_LOCALRANKID is only valid on a single-node cluster. In a multi-node cluster
```

Comment:
Is `PMI_RANK` always available in different MPI implementations such as `OpenMPI`, `MPICH`?

Reply:
Yes. it is as per the Intel's documentation for the MPI support.